### ISSUE TITLE
Feature Request: better mouse/touch support when no keyboard is accessible

### ISSUE THREAD

**Issue by robochase6000** [aseprite/aseprite#881]
been using aseprite on a surface pro and mostly loving it.  it seems designed to be used in conjunction with a keyboard, and i wish it were less reliant on a keyboard.  some suggestions:

1. give the toolbar more buttons so it's easier to select tools when you don't have hotkeys accessible.  having to constantly click between the eraser and eyedropper is really tedious, for example. having separate buttons for zoom and grab would make navigating around a big canvas a lot easier too.
2. i haven't been able to figure out how to re order layers with just the mouse.  i think you need to press some button on the keyboard when you're dragging layers around?  it would be great to have some mouse friendly way of doing this.  i would settle for up/down arrows on the layer window that just move the current layer up/down.
3. implementing some of these gestures would be awesome, but i'm sure it's complicated - https://helpx.adobe.com/photoshop/using/touch.html

Thanks for taking a look!

**Comment by KasumiArai** [comment]
About 2: To move a layer, click the layer, then click and drag the yellow border. Clicking and dragging on the layers window allows you to select multiple layers to move, so it's clicking and dragging the yellow border that actually moves them.

**Comment by dacap** [comment]
Hi! As @KasumiArai, you can drag the selection border to drag layers and frames: http://www.aseprite.org/docs/move-layers/

Anyway I suppose that it's quite hard to drag the borders when you are using your finger/a touch device.

**Comment by robochase6000** [comment]
oh cool, i'll try that out.  i'm using a stylus so it's a lot more accurate than a touch, but it's shakey enough where pixel hunting can be a little troublesome. thanks for the tip

**Comment by robochase6000** [comment]
regarding that doc page you linked to - i completely missed that, despite going to http://www.aseprite.org/docs/layers/ ...there ought to be an obvious link on that page to avoid further confusion.

**Comment by dacap** [comment]
@robochase6000 yeah, you're right, I'll add a link in Layers page to Move Layers
